### DISCUSSION TITLE
How sustainable is django-joplin_vieweb

### DISCUSSION THREAD

**Discussion by krueuw** [gri38/django-joplin_vieweb#70] [General]
Hello gri38, I am very pleased to see that there is (again) a possibility to display Joplin notes in the browser. How sustainable is your approach? If there is a version 3 of Joplin that contains significant changes, are you ready and able to follow them, or is there a risk that this will peter out again? In short, does it make sense to invest the effort because you will be working permanently on the topic?

**Comment by gri38** [comment]
Hi.

I use joplin daily on the web (via this project), so I'll make it evolve with Joplin.
I'm quite confident about the sustainability of my approach because it relies on [Joplin terminal](https://joplinapp.org/terminal/) and the [joplin API](https://joplinapp.org/api/references/rest_api/). So as long as these tools are provided by the team that releases Joplin, I will be able to follow their evolutions quite easily.

By the way: I've developped what is usefull for me, for instance I don't use todos, only checkboxes on notes. If someone would like some feature, don't hesitate, I'll do what I can (it's more a matter of time, between my jobs my kids and my mountain bike ;-) )

**Reply by krueuw** [reply]
I hope for the best and wish strength and time to keep things going :-) Then I will start working on setting up the software.
